models/Group.model.js

```diff
@@ -0,0 +1,33 @@
+const { Schema, model } = require("mongoose");
+
+const groupSchema = new Schema(
+  {
+    groupName: {
+      type: String,
+      require: true,
+      unique: true
+    },
+    description: {
+      type: String,
+    },
+    members: [
+      {
+        type: Schema.Types.ObjectId,
+        ref: "User"
+      },
+    ],
+    movies: [
+      {
+        type: Schema.Types.ObjectId.apply,
+        ref: "Movie"
+      }
+    ]
+  },
+  {
+    timestamps: true
+  }
+);
+
+const Group = model("Group", userSchema);
```

Comment:
Just found an error here: 

```suggestion
const Group = model("Group", groupSchema);
```